observability/metrics/src/main/scala/com/daml/metrics/api/MetricsContext.scala

```diff
@@ -27,6 +27,9 @@ case class MetricsContext(labels: Map[String, String]) {
     */
   def merge(context: MetricsContext): MetricsContext = this.copy(labels = labels ++ context.labels)
 
+  def withLabels(extraLabels: (String, String)*): MetricsContext =
```

Comment:
I would change to `withExtraLabels` to better clarify that we add extra labels alongside the existing ones